Library/Formula/libguestfs.rb

```diff
@@ -0,0 +1,216 @@
+      url "https://gist.githubusercontent.com/shulima/2feb769e9fcbb2f7a84f/raw/2e99df4f5b16ab8a9624163b9c7d34407ce688f0/lcrypt.patch"
+      sha256 "f6cae96bd32bb20308c087f091ffd7ff9240bfe8d3a5ea029eeedbeb4a1aa73e"
+    end
+    patch do
+      # Check if POSIX_FADVISE exists before using it
+      url "https://gist.githubusercontent.com/shulima/c45bc24af8e0291bfb95/raw/44057e50b0566fb8526838d4927db2f4aa04510f/posix_fadvise.patch"
+      sha256 "86cd41ddfe85309a2d1c0cc7ffc37330f83f3f5be5fc894b4127ec65cbf52d73"
+    end
+    patch do
+      # Replace Linux-specific fuse commands
+      url "https://gist.githubusercontent.com/shulima/b1bfa6accd67c457d5c0/raw/8405558ed32897e5294675f5f2b4bd65990b6f3c/fuse-bsd.patch"
+      sha256 "cff6759d306077c199bc2ef9503957a0be15b87006e4a07eeb1ba725c5515059"
+    end
+    patch do
+      # Turn off doclint to allow Java bindings to compile
+      url "https://gist.githubusercontent.com/shulima/00735be5ece79da21b91/raw/c259d2076c2fc620052b72d71a48bb464b5a4e3d/java_doclint.patch"
+      sha256 "d32d895daa359111194485ed0d04ab29251952e9395235e17a8426388aeea2c2"
+    end
+    patch do
+      # Add third parameter to xdrproc_t callbacks
+      url "https://gist.githubusercontent.com/shulima/65b6445698c4d61d7314/raw/1a873e3f70d6805f346d972bcf2ae734abb57000/xdrproc_t.patch"
+      sha256 "83aaa05ab8e348ca740cacd43faa51ad3b20c80441da35d10f222f6a6b27952f"
+    end
+    patch do
+      # Use getprogname where available
+      url "https://gist.githubusercontent.com/shulima/b232a15b8ff877f817bd/raw/4e301e4e7a02d3aa27f3c2fb5faee69c1c39f3b2/getprogname.patch"
+      sha256 "6aafa3718b73c45124c72357b460ddc91accc0a418e830a38a933f43cd2feac2"
+    end
+    patch do
+      # Add MacOSX-specific byteswap defines
+      url "https://gist.githubusercontent.com/shulima/5ad4c6fbfdfd27048fe8/raw/aff4eaf1c092af50e9b4af46fca352f743bbc9a1/byteswap.patch"
+      sha256 "e653c59b38cbe2b77cd0271603007db0c2973d00fe666cfbb42732eeade1ca28"
+    end
+    patch do
+      # Define SOCK_CLOEXEC and SOCK_NONBLOCK
+      url "https://gist.githubusercontent.com/shulima/3ca0eed701cc27bfbc71/raw/308405bd49188d414caf1198d296d37060c02777/sock_defines.patch"
+      sha256 "bca242cba0ceb73852007e75f83437d081ae7323790928eb0f0be63de3b33afe"
+    end
+    patch do
+      # Look up correct extension for ruby libs
+      url "https://gist.githubusercontent.com/shulima/c42dbaadd26535725666/raw/c86b370ec2797f0e3fe76a96c9852de6e015f586/ruby_dlext.patch"
+      sha256 "e3ed3c851c5f294bf3ff61aca64be8baff857740a8a7d842097bc9f3e140562a"
+    end
+    patch do
+      # Add DYLD_LIBRARY_PATH
+      url "https://gist.githubusercontent.com/shulima/c5fa3de7c84a0352e97e/raw/961e857c4201a82ad05126fbb17bd1f2912f5f43/dyld_library_path.patch"
+      sha256 "254d9186880cd17cdf478a514b327e372eebaed539df9b172dceb35021265e6c"
+    end
+  end
+
+  # The two required gnulib patches have been reported to gnulib mailing list, but with little effect so far.
```

Comment:
In the formula, or here?

http://lists.gnu.org/archive/html/bug-gnulib/2015-02/msg00083.html
http://lists.gnu.org/archive/html/bug-gnulib/2015-02/msg00078.html